### ISSUE TITLE
Error "Could not resolve the method in class Landroid/view/View" during deodexing

### ISSUE THREAD

**Issue by bursoft-git** [JesusFreke/smali#481]
There is another error of deodexing file from official SONY Xperia M4 Aqua firmware with baksmali-2.2b4.jar and latest commit https://github.com/JesusFreke/smali/commit/1a83d5a2d052e7e5896741807e855e99668f755a
```
java -jar baksmali-2.2.jar x -b boot.oat GmsCore.odex/system/priv-app/GmsCore/GmsCore.apk:classes3.dex
org.jf.dexlib2.analysis.AnalysisException: Could not resolve the method in class Landroid/view/View; at index 726
        at org.jf.dexlib2.analysis.MethodAnalyzer.analyzeInvokeVirtualQuick(MethodAnalyzer.java:1864)
        at org.jf.dexlib2.analysis.MethodAnalyzer.analyzeInstruction(MethodAnalyzer.java:1021)
        at org.jf.dexlib2.analysis.MethodAnalyzer.analyze(MethodAnalyzer.java:201)
        at org.jf.dexlib2.analysis.MethodAnalyzer.<init>(MethodAnalyzer.java:131)
        at org.jf.baksmali.Adaptors.MethodDefinition.addAnalyzedInstructionMethodItems(MethodDefinition.java:463)
        at org.jf.baksmali.Adaptors.MethodDefinition.getMethodItems(MethodDefinition.java:371)
        at org.jf.baksmali.Adaptors.MethodDefinition.writeTo(MethodDefinition.java:238)
        at org.jf.baksmali.Adaptors.ClassDefinition.writeVirtualMethods(ClassDefinition.java:326)
        at org.jf.baksmali.Adaptors.ClassDefinition.writeTo(ClassDefinition.java:112)
        at org.jf.baksmali.Baksmali.disassembleClass(Baksmali.java:152)
        at org.jf.baksmali.Baksmali.access$000(Baksmali.java:46)
        at org.jf.baksmali.Baksmali$1.call(Baksmali.java:76)
        at org.jf.baksmali.Baksmali$1.call(Baksmali.java:74)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
opcode: invoke-virtual-quick/range
code address: 8
method: Lugo;->a(ILandroid/view/View;Landroid/view/ViewGroup;Z)Landroid/view/View;
```

Necessairy files here https://yadi.sk/d/Z-cYda4U37wgLD

**Comment by JesusFreke** [comment]
Hmm. When I try this with tip-of-tree (1a83d5a) it works fine.

**Comment by bursoft-git** [comment]
This error affects only for classes3.dex, other classes deodexed fine. I also check commit https://github.com/JesusFreke/smali/commit/961c21be988e842cb552e8e55f59b69656bffc6c , same error.

**Comment by JesusFreke** [comment]
Oh, I had missed that. Thanks!

**Comment by JesusFreke** [comment]
This should be fixed by 9e3e86fda0eeb429c81de5123716d0fc5c9a0a5d
